### PR TITLE
Base layout & Footer Component

### PR DESCRIPTION
**Pull request by Gumbosh**
- Footer component added
- Base Layout added

### CODE REVIEW

**Comment by marcinstenka** [comment]
I would change phone number to 664182880, rest looks good!

File: src/components/Footer.astro [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+<div class="text-gray">
+  <Image src={logo} alt="logo" loading="lazy" width="100" />
+  <p class="mt-[15px]">
+    A perfect beauty salon near you <br /> to make you more beautiful
+  </p>
+  <address class="flex flex-col mt-[15px]">
+    <a
+      class="hover:text-accent"
+      href="tel:333333333"
+      aria-label="phone number">+48 333 333 333</a
```
Comment (MateuszBukowski93) [inline src/components/Footer.astro]: numer teleofnu do poprawy

Comment (MateuszBukowski93) [reply src/components/Footer.astro]: w dodatku on pewnie bedzie pojawiał się pare razy, wiec telefon i mail możesz dać do jakiegoś pliku constants i je importowac :)

**Comment by AleksandraWojcikk** [comment]
If you added a new component 'Footer' so you can remove <footer...> from layout.astro file and only import Footer as the component there, I think... Now it is duplicated.

**Review by marcinstenka — APPROVED** [review]

**Comment by Gumbosh** [comment]
- File for constant variables added
- Refactored files across application to use phone number declared in constants.ts
- Commented out email adress from footer component

**Comment by AleksandraWojcikk** [comment]
E-mail address is still visible on the website - because there are no changes in <footer...> in Layout.astro. As I mentioned above I think if now we have the Footer component it should be used there.

**Comment by Gumbosh** [comment]
> E-mail address is still visible on the website - because there are no changes in <footer...> in Layout.astro. As I mentioned above I think if now we have the Footer component it should be used there.

Layout.astro will be deleted shortly. For now, its purpose is templating.

**Comment by Gumbosh** [comment]
> I would change phone number to 664182880, rest looks good!

Done 👍🏻

**Review by AleksandraWojcikk — APPROVED** [review]

File: src/components/Footer.astro (hunk 1 above)
Comment (Gumbosh) [reply src/components/Footer.astro]: Telefon poprawiony, plik na stałe zmienne stworzony.

**Pull request merged**